packages/plugin-core/src/components/lookup/LookupControllerV3.ts

```diff
@@ -141,27 +144,33 @@ export class LookupControllerV3 {
       `Lookup (${this.nodeType})`,
       `- version: ${DendronWorkspace.version()}`,
     ].join(" ");
-    provider.provide(this);
+    Logger.info({ ctx, msg: "exit" });
     return { quickpick };
   }
 
   async showQuickPick(opts: ShowQuickPickOpts) {
+    const ctx = "showQuickPick";
+    Logger.info({ ctx, msg: "enter" });
     const cancelToken = this.createCancelSource();
     const { nonInteractive, provider, quickpick } = _.defaults(opts, {
       nonInteractive: false,
     });
+    Logger.info({ ctx, msg: "onUpdatePickerItems:pre" });
     // initial call of update
     await provider.onUpdatePickerItems({
       picker: quickpick,
       token: cancelToken.token,
       fuzzThreshold: this.fuzzThreshold,
     });
+    Logger.info({ ctx, msg: "onUpdatePickerItems:post" });
     if (!nonInteractive) {
+      provider.provide(this);
```

Comment:
Does moving this line add any perceivable delay between typing "Ctrl+L" and focus moving to the input box?

Reply:
I just did a quick profile and this line takes 0ms in our workspace 😄  `provider.provide` is mostly just binding listener methods so I imagine there won't be any perceivable delay.